recipe/session/session_utils_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2024, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
name of this file should be sessionUtils_tests.go (see how other test files are named)